pom.xml

```diff
@@ -19,100 +19,93 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>nexus-plugins</artifactId>
-    <version>3.18.0-01</version>
+    <version>3.19.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>nexus-repository-helm</artifactId>
+  <artifactId>nexus-repository-base</artifactId>
+  <groupId>org.sonatype.nexus.plugins</groupId>
   <name>${project.groupId}:${project.artifactId}</name>
-  <version>0.0.12</version>
+  <version>1.0.0</version>
   <inceptionYear>2018</inceptionYear>
-  <packaging>bundle</packaging>
-
-  <dependencies>
-
-    <dependency>
-      <groupId>org.sonatype.nexus</groupId>
-      <artifactId>nexus-plugin-api</artifactId>
-      <scope>provided</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.sonatype.nexus</groupId>
-      <artifactId>nexus-repository</artifactId>
-      <scope>provided</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.apache.commons</groupId>
-      <artifactId>commons-compress</artifactId>
-      <scope>provided</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.sonatype.goodies</groupId>
-      <artifactId>goodies-testsupport</artifactId>
-      <scope>test</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.spockframework</groupId>
-      <artifactId>spock-core</artifactId>
-      <scope>test</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.sonatype.nexus</groupId>
-      <artifactId>nexus-rapture</artifactId>
-      <scope>provided</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.yaml</groupId>
-      <artifactId>snakeyaml</artifactId>
-      <version>1.20</version>
-    </dependency>
-
-  </dependencies>
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>nexus-repository-helm</module>
+    <!-- Uncomment this when adding adding IT -->
+    <!--<module>nexus-repository-helm-it</module>-->
+  </modules>
+
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+  </distributionManagement>
+
+  <properties>
+    <nxrm-version>3.19.0-SNAPSHOT</nxrm-version>
```

Comment:
If we could find some way to use this <nxrm-version> value in Dockerfile, etc, that would go a long way to removing duplication.

Maybe something equivalent to this to use maven to read the project version (for use elsewhere) ?:

          mvn build-helper:parse-version versions:set -DnewVersion=\${parsedVersion.majorVersion}.\${parsedVersion.minorVersion}.\${parsedVersion.nextIncrementalVersion} versions:commit
          MVN_VERSION=$(mvn -q -Dexec.executable="echo" -Dexec.args='${project.version}' --non-recursive exec:exec)

Reply:
There definitely should be a more concise way to make Docker read properties from `pom.xml`. I will check the options available later